src/server/pfs/fuse/loopback.go

```diff
@@ -621,25 +623,32 @@ func (n *loopbackNode) trimTargetPath(path string) string {
 	return strings.TrimPrefix(path, "/")
 }
 
-func (n *loopbackNode) branch(repo string) string {
+func (n *loopbackNode) branch(name string) string {
 	// no need to lock mu for branches since we only ever read from it.
-	if branch, ok := n.root().branches[repo]; ok {
+	if branch, ok := n.root().branches[name]; ok {
 		return branch
 	}
 	return "master"
 }
 
-func (n *loopbackNode) commit(repo string) (string, error) {
+func (n *loopbackNode) commit(name string) (string, error) {
 	if commit, ok := func() (string, bool) {
 		n.root().mu.Lock()
 		defer n.root().mu.Unlock()
-		commit, ok := n.root().commits[repo]
+		commit, ok := n.root().commits[name]
 		return commit, ok
 	}(); ok {
 		return commit, nil
 	}
-	branch := n.root().branch(repo)
-	bi, err := n.root().c.InspectBranch(repo, branch)
+	ro, ok := n.root().repoOpts[name]
+	if !ok {
+		// not ready or something?
+		// XXX should this be an error?
```

Comment:
I think it would make sense to return an error (or maybe even just log an error) here.

Reply:
fixed